hr_attendance_rest_time_included/models/hr_attendance.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 Tecnativa - Eduardo Ezerouali
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import api, fields, models
+
+
+class HrAttendance(models.Model):
+    _inherit = "hr.attendance"
+
+    hr_attendance_rest_time = fields.One2many(
```

Comment:
This is inside `hr.attendance`, so the prefix `hr_attendance` is useless, redundant and takes a lot of space, and on the other side, use `_ids` suffix convention for o2m:

```suggestion
    rest_time_ids = fields.One2many(
```